component/loki/process/internal/stages/label_drop.go

```diff
@@ -0,0 +1,46 @@
+package stages
+
+// This package is ported over from grafana/loki/clients/pkg/logentry/stages.
+// We aim to port the stages in steps, to avoid introducing huge amounts of
+// new code without being able to slowly review, examine and test them.
+
+import (
+	"errors"
+	"time"
+
+	"github.com/prometheus/common/model"
+)
+
+// ErrEmptyLabelDropStageConfig error returned if config is empty.
+const ErrEmptyLabelDropStageConfig = "labeldrop stage config cannot be empty"
```

Comment:
I know we just ported this over, but it's weird to have an error variable be a string. This should really be an error type instead of wrapping it in `errors.New()` every time it's returned.

Reply:
True, should have picked this up. Fixing it.